docs/cookbooks/multus/build/cni-dhcp-daemon/README.md

```diff
@@ -0,0 +1,16 @@
+# cni-dhcp-daemon
+
+This Docker image can be used to run the [DHCP daemon from the
+containernetworking
+project](https://github.com/containernetworking/plugins/tree/master/plugins/ipam/dhcp).
+
+If you need to build a new version of the image and push it to Dockerhub, you
+can run the following:
+
+```bash
+docker build -t antrea/cni-dhcp-daemon:latest .
+docker push antrea/cni-dhcp-daemon:latest
```

Comment:
Not sure if it is too much for us to maintain a DHCP daemon image to demonstrate Multus.

Reply:
I don't believe this will be a maintenance burden. I can always move the image to my personal Github / Dockerhub accounts, but I don't think this is a better solution. I couldn't find a good existing image anywhere either.

Reply:
Ok. No strong opinion on this.